Week-3/CSS-ADVANCED/q2.html

```diff
@@ -0,0 +1,175 @@
+oatmeal_cookies.html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+
+        body {
+        margin: 0;
+        font-family: Helvetica, sans-serif;
+        }
+
+        header {
+        border-bottom: 2px solid #93B5C6;
+        background-color: #DDEDAA;
+        padding: 8px;
+        display: flex;
+        justify-content: space-between;
+        }
+
+        .logo {
+        color: #BD4F6C;
+        font-size: 18px;
+        }
+
+        .content {
+        margin-left: 30px;
+        display: flex;
+        flex-wrap: wrap;
+        }
+
+        .title {
+        border-bottom: 1px dashed #93B5C6;
+        display: flex;
+        align-items: baseline;
+        }
+
+        .title h1 {
+        color: #BD4F6C;
+        }
+
+        .title p {
+        color: #93B5C6;
+        margin-left: 30px;
+        display: flex;
+        align-items: baseline;
+        }
+
+        .summary {
+        margin-top: 30px;
+        display: flex;
+        justify-content: space-evenly;
```

Comment:
Be cautious with using` justify-content: space-evenly;` as it is not supported in all browsers. You could consider using `justify-content: space-around;` for better compatibility.